provisioning/device/src/ExponentialBackoff.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
```

Comment:
Missing copyright and license information at file beginning

Reply:
Good catch, will add